### PR TITLE
Update instance panel with new layer

### PR DESCRIPTION
**Pull request by AlexandreSi**
This PR make it so that instance properties panel is force updated when a layer is created or renamed.

### CODE REVIEW

File: newIDE/app/src/SceneEditor/index.js [hunk 1]
```diff
@@ -674,6 +674,11 @@ export default class SceneEditor extends React.Component<Props, State> {
   ) => {
     this.props.initialInstances.moveInstancesToLayer(oldName, newName);
     done(true);
+    this.forceUpdatePropertiesEditor();
```
Comment (ClementPasteau) [inline newIDE/app/src/SceneEditor/index.js:677]: nice! should this be before the done?

**Review by 4ian — APPROVED** [review]
Lgtm!

**Review by D8H — APPROVED** [review]

**Review by ClementPasteau — APPROVED** [review]
nice!

File: newIDE/app/src/SceneEditor/index.js (hunk 1 above)
Comment (AlexandreSi) [reply newIDE/app/src/SceneEditor/index.js:677]: Nope, this callback is what actually renames the layer after all instances has been updated with the new layer name

**Pull request merged**